docs/installation.md

```diff
@@ -1,101 +1,50 @@
-
-* domain-protect
-  * main.tf
-  * variables.tf
-  * terraform.tfvars
-  * build
-  * lambda_code
-  * scripts
-    * lambda-build
-      * create-package-for-each.sh
-      * create-package.sh
-
-Finally, make sure you the scripts in `scripts/lambda-build` are executable before committing to your repo by running `chmod +x`
-
-The downside of this approach is changes to `build`, `lambda_code`, or `scripts` folders from source folder have to be synced manually to local folder
+| VARIABLE                        | EXAMPLE VALUE / COMMENT                               |
+| ------------------------------- | ------------------------------------------------------|
+| environment                     | "dev" (not needed if Terraform workspace used)        |
+| org_primary_account             | "012345678901"                                          |  
+| security_audit_role_name        | "dp-audit" (not needed if "domain-protect-audit" used)|
+| slack_channels                  | ["security-alerts-dev"]                               |
+| slack_webhook_urls              | ["https://hooks.slack.com/services/XXX/XXX/XXX"]      | 
 
-This is only one of the many ways to deploy with atlantis. You should tailor to your environment. Please reach out to us to brainstorm on your deployment workflow.
+* Add extra variables if desired as detailed in [Module imputs](https://registry.terraform.io/modules/domain-protect/domain-protect/aws/latest?tab=inputs)
+* see the [Examples directory](https://github.com/domain-protect/terraform-aws-domain-protect/tree/main/examples) for complete Terraform examples including `provider.tf` and `backend.tf` files 
 
-## Manual installation (not recommended)
+## Multiple environments
+Domain Protect is designed so that multiple environments can be deployed, e.g. `dev` and `prd`.
 
-* replace the Terraform state S3 bucket fields in the command below as appropriate
-* for local testing, duplicate `terraform.tfvars.example`, rename without the `.example` suffix
-* enter details appropriate to your organization and save
-* alternatively enter Terraform variables within your CI/CD pipeline
-* deploy development environment for detection only
-* default scan schedule for dev environment is 12 hours
+It's important that only one environment, e.g. `prd` can perform active takeover, to avoid conflicts between environments.
 
-```bash
-terraform init -backend-config=bucket=TERRAFORM_STATE_BUCKET -backend-config=key=TERRAFORM_STATE_KEY -backend-config=region=TERRAFORM_STATE_REGION
-terraform workspace new dev
-terraform plan
-terraform apply
-```
+* ensure you only set the variable `takeover = true` for a single environment, e.g. `prd`
 
-* deploy production environment for detection and automated takeover
-* default scan schedule for `prd` environment is 60 minutes
+Make sure to also update `production_environment` to match the `environment` variable when deploying to production.
```

Comment:
This part is still a bit silly to me, not the documentation, but the necessity of this itself. I think we can remove this input entirely. We should create a ticket for future reference